habitat-lab/habitat/config/benchmark/rearrange/rearrange_easy_multi_agent.yaml

```diff
@@ -37,10 +37,7 @@ habitat:
     agent_0:
       radius: 0.3
       sim_sensors:
```

Comment:
I think this line should be removed or we should add the sensors. If not we get the following error:

```
hydra.errors.ConfigCompositionException: In 'rearrange_easy_multi_agent.yaml': ValidationError raised while composing config:
field 'habitat.simulator.agent_0.sim_sensors' is not Optional
```

Reply:
Now fixed.